_includes/posts/maintenance-table.md

```diff
@@ -1,7 +1,7 @@
-| 28.x   | TBA*       | after v30.0 | after v31.0  |
+| 28.x   | 2024-10-02 | after v30.0 | after v31.0  |
```

Comment:
nit: for consistency, would keep the current format?
```suggestion
| 29.x   | TBA*       | after v31.0 | after v32.0  |
| 28.x   | 2024-10-02 | after v30.0 | after v31.0  |
```

Reply:
Done